### PR TITLE
Kolibri build tools fixes

### PR DESCRIPTION
**Pull request by rtibbles**
### Summary
Clean up from #4524.

* Do not ignore kolibri-tools and kolibri packages when doing code transformation in jest.
* Add all dependencies from `kolibri-core` when building the exported `kolibri` module for the core api.
* Make the `kolibri-tools test` command seamlessly pass through all command line args to jest.

### Reviewer guidance
Can you pass arbitrary command line args to jest?

Does this make sense?

If this is all fine, I'll do another dev release of the two npm modules. cc @ralphiee22 

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Gherkin stories have been updated
- [ ] Unit tests have been updated

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

File: jest.conf.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-module.exports = {
-  transformIgnorePatterns: ['/node_modules/(?!(keen-ui|epubjs)/).*/'],
+const baseConfig = require('kolibri-tools/jest.conf');
+
+module.exports = Object.assign(baseConfig, {
+  transformIgnorePatterns: ['/node_modules/(?!(keen-ui|epubjs|kolibri-tools|kolibri)/).*/'],
```
Comment (indirectlylit) [inline jest.conf.js:5]: maybe add a comment to this?

it's unclear to me why we'd be ignoring `kolibri-tools` and `kolibri`

File: packages/kolibri-tools/build.js [hunk 2]
```diff
@@ -20,3 +21,19 @@ __builder.buildApiSpec();
...
+
+const kolibriPackageFile = path.resolve(__dirname, '../kolibri/package.json');
+
+const kolibriPkg = require(kolibriPackageFile);
+
+kolibriPkg.dependencies = kolibriCorePkg.dependencies;
+
+fs.writeFileSync(kolibriPackageFile, JSON.stringify(kolibriPkg, undefined, 2), {
+  encoding: 'utf-8',
+});
```
Comment (indirectlylit) [inline packages/kolibri-tools/build.js:39]: wait .. so we're overwriting the package file, and then this gets checked into git?

I think this makes it unclear when to `yarn workspace kolibri-core add ...` and when to  `yarn workspace kolibri add ...`

File: jest.conf.js (hunk 1 above)
Comment (rtibbles) [reply jest.conf.js:5]: Can do - but this line is saying don't ignore these for code transpilation.

File: packages/kolibri-tools/build.js (hunk 2 above)
Comment (rtibbles) [reply packages/kolibri-tools/build.js:39]: The `kolibri-core` package is the only one that should be added to; `kolibri` is the version generated for export to an npm module. We're overwriting the package file in `packages/kolibri` in this line, can add more comments.

**Review by indirectlylit — APPROVED** [review]
Apart from the thread above and #4577, this works fine for me.

File: jest.conf.js (hunk 1 above)
Comment (indirectlylit) [reply jest.conf.js:5]: ah, missed the `!`

**Pull request merged**